### PR TITLE
CSS updates

### PR DESCRIPTION
**Pull request by skoolbus39**
Update book pages block menu styling:
- Updated padding, font-size, colors, background-colors
- Use Unicode glyphs instead of raster images

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -208,3 +208,9 @@ input.webform-calendar {
   float: left;
   margin: 0 20px 30px 0;
 }
+
+/* book pages */
+ul li.leaf {
```
Comment (ericras) [inline css/style.css]: Add ul li.expanded

File: css/menu-block.css [hunk 2]
```diff
@@ -3,25 +3,28 @@
...
+.block-menu-block li {padding: 0; line-height: 1.333;}
+.block-menu-block ul a {border: none; font-size: .75rem}
 .block-menu-block ul:first-child > li,
 .block-menu-block ul:first-child > li > ul > li {list-style-image: none; list-style-type: none;}
-.block-menu-block ul:first-child > li > a {display: block; background-color: #f2edde; padding: .5em .5em .5em 1.5em; margin-bottom: 1px; border-bottom: 1px solid #E2E2E2;}
-.block-menu-block ul:first-child > li > a {background-position: .3em center; background-repeat: no-repeat;}
-.block-menu-block ul:first-child > li > ul > li > a {background-position: 1em center; background-repeat: no-repeat;}
-.block-menu-block li.collapsed > a {background-image: url("../../../../../misc/menu-collapsed.png");}
-.block-menu-block li.expanded > a {background-image: url("../../../../../misc/menu-expanded.png");}
+.block-menu-block ul:first-child > li > a {display: block; padding-left: 2.532em; background-color: #eae9e6;}
```
Comment (ericras) [inline css/menu-block.css]: Move this below the next rule.  THe padding-left is getting overridden

File: css/menu-block.css [hunk 3]
```diff
@@ -3,25 +3,28 @@
  * book navigation block in unl_fourone_preprocess_block().
  */
 
-.block-menu-block a {color: #333;}
-.block-menu-block a:hover {color: #d00000;}
-.block-menu-block a.active {background-color: #F2E8B1 !important;}
```
Comment (ericras) [inline css/menu-block.css:8]: I think it would be nice to have some sort of visual indicator on .active

File: css/menu-block.css [hunk 4]
```diff
@@ -3,25 +3,28 @@
...
-.block-menu-block h3 {background-color: #454542; color: #FFF; font-size: inherit; text-transform: uppercase; margin: 0; padding: 1em;}
-.block-menu-block h3 a {color: #fff; border: none;}
+.block-menu-block a {padding-right: 1.777em; color: #474746;}
+.block-menu-block a:hover {cursor: pointer; color: #d00000;}
+.block-menu-block a.active {color: #6a0000;}
+.block-menu-block h3 {background-color: #474746; color: #fefdfa; font-size: .802rem; text-transform: uppercase; margin: 0; line-height: 1.333}
+.block-menu-block h3 a {display: block; padding-top: 1.333em; padding-bottom: 1.069em; padding-left: 1em; border: none; color: #fefdfa;}
```
Comment (ericras) [inline css/menu-block.css]: Move padding to the h3 for the use case where the header isn't linked (an option in Menu Block)

File: css/style.css (hunk 1 above)
Comment (ericras) [inline css/style.css]: This wipes out list-style for all drupal lists but this pull request only fixes the list style for .block-menu-block

This should be moved to the other file and scoped to .block-menu-block

File: css/menu-block.css (hunk 3 above)
Comment (skoolbus39) [inline css/menu-block.css:8]: I agree and will explore options for the active state.

File: css/menu-block.css [hunk 5]
```diff
@@ -3,25 +3,36 @@
...
+.block-menu-block li.leaf > a {position: relative;}
+.block-menu-block li.collapsed > a:before,
+.block-menu-block li.expanded > a:before,
+.block-menu-block ul > li > ul > li > ul li.leaf > a:before {position: absolute; top: 50%; transform: translateY(-50%); margin-left: -.75rem; font-size: .451rem; color: #5b5b5a;}
+.block-menu-block li.collapsed > a:before,
+.block-menu-block li.expanded > a:before {font-size: .451rem;}
+.block-menu-block li.collapsed > a:before {content: '\25B6';}
+.block-menu-block li.expanded > a:before {content: '\25BC';}
+.block-menu-block ul > li > ul > li > ul li.leaf > a:before {content: '\2022';}
+.block-menu-block ul li.leaf,
```
Comment (ericras) [inline css/menu-block.css:37]: looks like we need "ul li.collapsed" here as well

**Pull request merged**